src/shared_modules/utils/stringHelper.h

```diff
@@ -310,7 +305,43 @@ namespace Utils
 
         return !str.empty() && it == str.end();
     }
-}
+
+    static int parseStrToBool(const std::string& str)
+    {
+        return (str.compare("yes") == 0) ? 1 : (str.compare("no") == 0) ? 0 : -1;
+    }
+
+    static long parseStrToTime(const std::string str)
+    {
+        std::size_t pos;
+        try
+        {
+            auto seconds {std::stoll(str, &pos)};
+
+            if (seconds < 0)
+            {
+                return -1;
+            }
+
+            switch (str.at(++pos))
```

Comment:
The length of "str" should also be checked in order to match the pattern "a number + only one letter".
Because if your string has a valid first letter after the number, but there are more characters, I think it should be considered an invalid string. For example "10winvalid_characters"

Reply:
Fix it =)